### PR TITLE
(node/lsstcam-ccobwb.cp.lsst.org) deploy host as el9

### PR DESCRIPTION
**Pull request by dtapiacl**

### CODE REVIEW

**Review by glennmorris — APPROVED** [review]

File: hieradata/node/lsstcam-ccobwb.cp.lsst.org.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  eno1:
+    content:
+      connection:
+        id: "eno1"
+        uuid: "03da7500-2101-c722-2438-d0d006c28c73"
+        type: "ethernet"
+        interface-name: "eno1"
+      ethernet: {}
+      ipv4:
+        address1: "139.229.175.81/26,139.229.175.126"
```
Comment (jhoblitt) [inline hieradata/node/lsstcam-ccobwb.cp.lsst.org.yaml:12]: The device association appears to be missing.
image

File: hieradata/node/lsstcam-ccobwb.cp.lsst.org.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    content:
+      connection:
+        id: "enp6s0"
+        uuid: "da97db8e-6400-4cca-9334-71995eb2ac1d"
+        type: "ethernet"
+        interface-name: "enp6s0"
+      ethernet: {}
+      ipv4:
+        method: "manual"
+        address1: "169.254.5.10/24"
```
Comment (jhoblitt) [inline hieradata/node/lsstcam-ccobwb.cp.lsst.org.yaml:29]: What is connected to this interface?

File: hieradata/node/lsstcam-ccobwb.cp.lsst.org.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    content:
+      connection:
+        id: "enp7s0"
+        uuid: "da97db8e-6400-4cca-9334-71995eb2ac1d"
+        type: "ethernet"
+        interface-name: "enp7s0"
+      ethernet: {}
+      ipv4:
+        method: "manual"
+        address1: "192.168.1.1/24"
```
Comment (jhoblitt) [inline hieradata/node/lsstcam-ccobwb.cp.lsst.org.yaml:43]: image

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: hieradata/node/lsstcam-ccobwb.cp.lsst.org.yaml (hunk 1 above)
Comment (dtapiacl) [reply hieradata/node/lsstcam-ccobwb.cp.lsst.org.yaml:12]: Device is registered on ipam.cp.lsst.org.

File: hieradata/node/lsstcam-ccobwb.cp.lsst.org.yaml (hunk 2 above)
Comment (glennmorris) [reply hieradata/node/lsstcam-ccobwb.cp.lsst.org.yaml:29]: > What is connected to this interface?

There is some Keysight device on one of these. (I would "at" youtsumi, but I guess it doesn't work for people who are not member of this repo.)
Ref https://lsstc.slack.com/archives/C06V270ECBW/p1716826028801209

Comment (glennmorris) [reply hieradata/node/lsstcam-ccobwb.cp.lsst.org.yaml:29]: Yes, this one is a direct connection to some Keysight something, ref
https://lsstc.slack.com/archives/C06V270ECBW/p1715976340596719?thread_ts=1715953274.383569&cid=C06V270ECBW
and replies

**Review by jhoblitt — APPROVED** [review]

**Pull request merged**